services/github/project/api/routes.py

```diff
@@ -200,7 +210,30 @@ def get_items():
                     item['reviewer_points'] = 0
                 item['num_reactions'] = r.get('reactions').get('totalCount') or 0
                 item['title'] = r.get('title')
-                items.append(item)
+                # check if date is between start and end date
+                if start_creation_date != None and end_creation_date != None:
+                    try:
+                        item_date = datetime.datetime.strptime(item['createdAt'].split('T')[0], "%Y-%m-%d")
+                        start_date = datetime.datetime.strptime(start_creation_date, "%Y-%m-%d")
+                        end_date = datetime.datetime.strptime(end_creation_date, "%Y-%m-%d")
+
+                        if check_between_dates(start_date, end_date, item_date):
+                            items.append(item)
+                    except:
+                        print("date format error, should be YYYY-MM-DD")
```

Comment:
For the exceptions, we need to return them through the API, [like so](https://github.com/sendgrid/dx-automator/blob/development/services/tasks/project/api/routes.py#L146).

Reply:
Please catch the specific exception, which I believe is a [`ValueError`](https://docs.python.org/2/library/datetime.html#date-objects), but please check in your tests.